applications/ConvectionDiffusionApplication/custom_utilities/move_particle_utility.h

```diff
@@ -271,7 +271,7 @@ namespace Kratos
 		virtual ~MoveParticleUtilityScalarTransport()
 		{}
 
-		void MountBin()
+		void MountBin(const bool& rCellSizeProvided=false, const double& rCellSize=1.0)
```

Comment:
```suggestion
		void MountBin(
		    const bool rCellSizeProvided=false,
		    const double rCellSize=1.0)
```
It is customary to pass simple types as copy. Also minor style guide suggestion.
Besides this, I think that we could only use the `rCellSize` parameter defaulted to zero to then check `if (rCellSize > 0.0)`.

Reply:
I created a new method with a cell size as argument, instead of modifying the existing one